package.json

```diff
@@ -25,11 +25,11 @@
     "build:script:projects": "lerna run build:script --scope=@tps/apps-projects",
     "build:script:rewards": "lerna run build:script --scope=@tps/apps-rewards",
     "clean:aragon": "rm -rf ~/.aragon && rm -rf ~/.ipfs",
-    "clean:build": "git clean -ffdx --exclude=node_modules/",
+    "clean:build": "npm run clean -- --exclude=node_modules/",
     "clean:commit": "rm -f package-lock.json && npm run bootstrap",
-    "clean": "git clean -ffdx",
-    "coverage": "lerna run coverage --no-bail",
-    "coveralls": "cat apps/*/coverage/lcov.info | coveralls",
+    "clean": "git clean -fXd",
+    "coverage": "./node_modules/.bin/lerna run coverage --no-bail",
+    "coveralls": "cat apps/*/coverage/lcov.info | ./node_modules/.bin/coveralls",
```

Comment:
Good stuff. Should we put it all in its own PR?

Reply:
Why not? I'll do it